backtester/engine/setup.go

```diff
@@ -865,3 +879,52 @@ func loadLiveData(cfg *config.Config, base *gctexchange.Base) error {
 	}
 	return nil
 }
+
+// ExecuteStrategy executes the strategy using the provided configs
+func ExecuteStrategy(strategyCfg *config.Config, backtesterCfg *config.BacktesterConfig) error {
+	if err := strategyCfg.Validate(); err != nil {
+		log.Error(log.Global, err)
+		return err
+	}
+	if backtesterCfg == nil {
+		err := fmt.Errorf("%w backtester config", common.ErrNilArguments)
+		log.Error(log.Global, err)
```

Comment:
Are these logs necessary, we could probably just log in to the calling function if needed, which might be better for clarity?